### ISSUE TITLE
[BUG] Unable to parse PostgreSQL function correctly

### ISSUE THREAD

**Issue by renathoaz** [cweijan/vscode-mysql#1365]
Identifier
cweijan.vscode-postgresql-client2
Version
8.0.6

- Database type and version: Postgres 15.8 (Debian 15.8-1.pgdg120+1)

Hi there, my functions has the definition volatile in the end,  newer versions started to complain about it:

![image](https://github.com/user-attachments/assets/4ec385e0-db76-4818-9549-9d9d5da5c452)

This works:

```
CREATE OR REPLACE FUNCTION get_random_value()
RETURNS INTEGER
LANGUAGE plpgsql VOLATILE
AS $$
BEGIN
    RETURN floor(random() * 100 + 1)::INTEGER;
END;
$$;

```
This doesn't, but running it directly in the database it's fine, it doesn't work in  vscode extension:

```
CREATE OR REPLACE FUNCTION get_random_value()
RETURNS INTEGER
AS $$
BEGIN
    RETURN floor(random() * 100 + 1)::INTEGER;
END;
$$ LANGUAGE plpgsql VOLATILE;
```

**Comment by cweijan** [comment]
I'm sorry, I will fix it in the next version. Before that, please move the language definition to the end of return.

<img width="604" alt="image" src="https://github.com/user-attachments/assets/ebd9b554-1122-421a-8796-ba8b4ec85b09" />

**Comment by cweijan** [comment]
Fixed in version 8.0.7.
